docs/install.md

```diff
@@ -0,0 +1,116 @@
+# Installing kpack
+
+* [Pre-requisites](#pre-requisites)
+* [Installing kpack](#installing-kpack)
+
+
+## Pre-requisites
+
+1. A Kubernetes cluster version 1.12 or later
+1. [kubectl cli](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+1. Cluster-admin permissions for the current user.
+1. Accessible Docker V2 Registry
+
+## Installing-kpack
+
+1. Download the most recent [github release](https://github.com/pivotal/kpack/releases).
+
+   ```bash
+   kubectl apply  --filename release.yaml
+   ```
+1. Ensure that the kpack controller has a status of `Running` using  `kubectl get`.   
+
+   ```bash
+   kubectl get pods --namespace kpack --wait
```

Comment:
`--wait` doesn't seem to work. I'll update kubectl and try again.

Reply:
Just updated, same thing. `--wait` should be removed

Reply:
--watch 

Good catch